px-map-behavior-marker-group.es6.js

```diff
@@ -672,25 +672,26 @@
       this._bindAndOpenPopup(evt.layer);
     },
 
-    _handleSingleMarkerDoubleTap(evt) {
+    _handleSingleMarkerDoubleClick(evt) {
       const latLng = L.latLng(evt.layer.getLatLng());
       const {lat, lng} = latLng;
       const detail = {
         latLng: latLng,
         lat: lat,
         lng: lng
       };
-      this.fire('px-map-marker-group-double-tapped', detail);
+      this.fire('px-map-marker-group-double-clicked', detail);
     },
     /**
-     * Fired when the marker is double clicked or tapped by the user.
+     * Fired when an individual marker is double clicked by the user. Not fired when a marker cluster is double clicked.
+     * Note that this will only work on non-touch (e.g. desktop) devices. Touch (e.g. mobile) devices cannot respond to double click/tap events.
      *
      *   * {Object} detail - Contains the event details
      *   * {Number} detail.lat - Latitude of the marker
      *   * {Number} detail.lng - Longitude of the marker
      *   * {L.LatLng} detail.latLng - Custom Leaflet object containing the lat and lng
      *
-     * @event px-map-marker-group-double-tapped
+     * @event px-map-marker-group-double-clicked
```

Comment:
Last thought on this.... lets make it more specific and do `px-map-marker-group-marker-double-clicked` to distinguish from a cluster being clicked, which could be implemented in the future. Weird long name, but it is specific.